community/contributing/bisecting_regressions.rst

```diff
@@ -105,16 +147,19 @@ Compile Godot. This assumes you've set up a build environment:
 
 .. code-block:: shell
 
-    # <platform> is the platform you're targeting for regression testing,
-    # like "windows", "x11" or "macos".
-    $ scons platform=<platform> -j4
+    $ scons
 
-Since building Godot takes a while, you want to dedicate as many CPU threads as
-possible to the task. This is what the ``-j`` parameter does. Here, the command
-assigns 4 CPU threads to compiling Godot.
+Run the engine
+^^^^^^^^^^^^^^
 
 Run the binary located in the ``bin/`` folder and try to reproduce the bug.
 
+.. note::
+
+    Double-check the output file name in ``bin/`` to make sure you're actually
+    running the binary you've just compiled. Different Godot versions will
+    output binaries with different names.
```

Comment:
Can also link to `/development/compiling/introduction_to_the_buildsystem.html#resulting-binary`. It currently doesn't have a reference tag, so that needs to be added as well.